chained/tlsfrag.go

```diff
@@ -0,0 +1,40 @@
+package chained
+
+import (
+	"net"
+	"strconv"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport/tlsfrag"
+	"github.com/getlantern/common/config"
+)
+
+func tlsFragConn(conn net.Conn, pc *config.ProxyConfig) net.Conn {
+	splitHello, index := splitHelloInfo(pc)
+	if splitHello {
+		if _, ok := conn.(*net.TCPConn); !ok {
+			return conn
+		}
```

Comment:
Wait does this work in a production context?  I thought we wrap the transport with our own type.

Reply:
Oh very good point....I wonder what it actually is there...will look.